src/components/Checkbox/Checkbox.module.css

```diff
@@ -7,158 +7,109 @@
 /**
  * A custom individual checkbox control
  * 1) Outer wrapper div for custom control
- * 2) Determines width and height of the control
+ * 2) Grid display aligns label and the checkbox
  * 3) Prevents control from being "squished" in 
  *    a flexbox parent
  */
-.checkbox {
+.checkbox { /* 1 */
   position: relative;
-  display: block;
-  width: var(--eds-size-2-and-half); /* 2 */
-  height: var(--eds-size-2-and-half); /* 2 */
-  flex: none; /* 3 */
+
+  display: grid; /* 2 */
+  align-items: flex-start;
+  grid-template-columns: min-content 1fr;
+  gap: var(--eds-size-1);
+}
+
+/**
+ * Text labelling the checkbox.
+ */
+.label {
+  color: var(--eds-theme-color-form-label-text);
+
+  /* Shift label so the checkbox always appears vertically centered
+  with the *first* line of text, in case label wraps. */
+  /* margin-top: calc((var(--checkbox-svg-size) - var(--label-line-height)) / 2); */
+}
+
+/**
+ * Size variants for the label.
+ * 1) line-height is set to be same as checkbox size to center label text relative to the checkbox.
+ */
+.label--sm {
+  @mixin eds-theme-typography-label-sm;
```

Comment:
these label tokens are smaller than the v0 label sizes for both small and medium sizes. Should the checkbox itself be smaller? (is [20px](https://github.com/chanzuckerberg/edu-design-system/pull/959/files#diff-204ed52cdcdeb2aade99c93d518d9783f564e65ff7df2d65976757fa27f03a31R81) right now), or is there a mock I couldn't find?

Reply:
Sounds good, will message Ashley